week3/week3_hw/Program.cs

```diff
@@ -0,0 +1,188 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "week-3 homework!");
+
+app.MapGet("/account", () =>
+{
+  var account = new Account(100);
+  account.Deposit(100);
+  Console.WriteLine(System.Text.Json.JsonSerializer.Serialize(account));
+  Console.WriteLine($"Account balance is {account.Balance}");
+  account.Withdraw(10);
+  Console.WriteLine(System.Text.Json.JsonSerializer.Serialize(account));
+  Console.WriteLine($"Account new balance is {account.Balance}");
+  Console.WriteLine(System.Text.Json.JsonSerializer.Serialize(account));
```

Comment:
it is best to return this as a result of the endpoint rather than writing it in the console.